addons/l10n_be/i18n/de.po

```diff
@@ -63,196 +56,144 @@ msgid "46 - Exempted intra-Community deliveries and ABC sales"
 msgstr "71 - Dem Staat geschuldete Steuer"
 
 #. module: l10n_be
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_72_formula
 #: model:account.report.line,name:l10n_be.tax_report_line_72
 msgid "72 - Amount owed by the State"
 msgstr "72 - Vom Staat geschuldete Summen"
 
 #. module: l10n_be
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_81_applied_carryover
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_81_balance
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_81_balance_unbound
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_81_carryover
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_81_tag
 #: model:account.report.line,name:l10n_be.tax_report_line_81
 msgid "81 - Trade goods, raw materials and consumables"
 msgstr "81 - Handelsgüter, Roh- und Hilfsstoffe"
 
 #. module: l10n_be
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_82_applied_carryover
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_82_balance
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_82_balance_unbound
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_82_carryover
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_82_tag
 #: model:account.report.line,name:l10n_be.tax_report_line_82
 msgid "82 - Services and miscellaneous goods"
 msgstr "82 - Leistungen und verschiedene Güter"
 
 #. module: l10n_be
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_83_applied_carryover
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_83_balance
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_83_balance_unbound
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_83_carryover
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_83_tag
 #: model:account.report.line,name:l10n_be.tax_report_line_83
 msgid "83 - Investment goods"
 msgstr "83 - Investitionsgüter"
 
 #. module: l10n_be
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_84_tag
 #: model:account.report.line,name:l10n_be.tax_report_line_84
 msgid "84 - Credit notes for operations in grids [86] and [88]"
-msgstr ""
-"84 - Gutschriften die sich auf Umsätze aus Raster [86] und [88] beziehen"
+msgstr "84 - Gutschriften für Umsätze aus Raster [86] und [88]"
 
 #. module: l10n_be
-#: model:account.report.expression,report_line_name:l10n_be.tax_report_line_85_tag
 #: model:account.report.line,name:l10n_be.tax_report_line_85
 msgid "85 - Credit notes received relating to other operations in part III"
 msgstr ""
-"85 - Gutschriften die sich auf die übrigen Umsätze aus Rahmen III beziehen"
+"85 - Erhaltene Gutschriften, die sich auf die übrigen Umsätze aus Rahmen III "
```

Comment:
FYI: All terms from the tax report came from the official document https://financien.belgium.be/sites/default/files/downloads/165-625-formular-2022.pdf. As long as the meaning is not different, I guess it's fine to simplify things. Just wanted to inform.